l10n/en-US/firefox/whatsnew/whatsnew-fx81.ftl

```diff
@@ -2,5 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+whatsnew81-our-latest-for-android-has = Our latest for { -brand-name-android } has lots of fun, smart and even weirdo features – like a moveable search bar. Because we’re an independent browser. We care about your thumbs.
```

Comment:
The weirdo comment was lost in the copy-paste.

Reply:
I don't think we need it for en-US, that was for localizers.